libmysqld/CMakeLists.txt

```diff
@@ -148,6 +148,10 @@ IF(WIN32)
   SET(MYSQLSERVER_OUTPUT_NAME mysqlserver)
   SET(COMPONENT_MYSQLSERVER "Embedded")
   SET(COMPONENT_LIBMYSQLD "Embedded")
+ELSEIF(CMAKE_SYSTEM_NAME MATCHES AIX)
+  SET(MYSQLSERVER_OUTPUT_NAME mariadbd_server)
```

Comment:
can you put a comment in the commit message as to why a different name was chosen here.

Reply:
Name conflict.
On AIX, static and shared libraries are both archive (.a file). In some cases, MariaDB uses the same name for different libraries (a static and another shared); it is OK on most OS because they have different extension. Not on AIX.
Solution can be to compile only one library (static or shared), or to change the name of one of them.